website/src/components/Chat/ChatListItem.tsx

```diff
@@ -78,7 +78,7 @@ export const ChatListItem = ({
     <Button
       // @ts-expect-error error due to dynamically changing as prop
       ref={rootRef}
-      {...(!isEditing ? { as: Link, href: ROUTES.CHAT(chat.id) } : { as: "div" })}
+      // {...(!isEditing ? { as: Link, href: ROUTES.CHAT(chat.id) } : { as: "div" })}
```

Comment:
Please revert this. Using a tag is much more better than JS

Reply:
Done.